docs/docs-content/clusters/edge/local-ui/cluster-management/link-hosts.md

```diff
@@ -0,0 +1,172 @@
+   hosts. The base-64 encoded token contains the IP address of the node, as well as an OTP that will expire in two
+   minutes. Once a token expires, the leader node generates another token automatically.
+
+4. Click the **Copy** button to copy the token.
+
+5. Log in to [Local UI](../host-management/access-console.md) on the node that you want to link to the leader node.
+
+6. From the left **Main Menu**, click **Linked Edge Hosts**.
+
+7. Click **Link this device to another**.
+
+8. In the pop-up box that appears, enter the token you copied from the leader node.
+
+9. Click **Confirm**.
+
+10. Repeat this process for every node you want to link to the leader node.
+
+### Validate
+
+1. Log in to [Local UI](../host-management/access-console.md) on the leader node.
+
+2. From the left **Main Menu**, click **Linked Edge Hosts**.
+
+3. Confirm that all nodes you linked together show up in the **Linked Edge Hosts** table.
+
+## Unlink Hosts
+
+You can unlink a host to either link it to another host or to use it for independent workloads. You can only unlink a
+follower host. Once all follower hosts have been unlinked, you can
+[remove its leader status](#remove-leader-node-status) and link it to another node.
+
+You can unlink a follower host on the follower host itself or from the leader node.
+
+### Prerequisites
+
+- Two or more linked hosts.
+
+- [Access to Local UI](../host-management/access-console.md) on the host you want to unlink or on the leader host.
+
+- The host you want to unlink cannot be in-use by a cluster. If you want to remove a node from your cluster and then
+  unlink it, you must first scale down the cluster. Once the host is in **Ready** state instead of **In-Use**, you can
+  proceed to unlink it. For more information, refer to [Scale Down a Cluster](./scale-cluster.md#scale-down-a-cluster).
+
+### Procedure
+
+<Tabs>
+
+<TabItem value="From Leader Node">
+
+1. Log in to Local UI on the leader node.
+
+2. From the left **Main Menu**, click **Linked Edge Hosts**.
+
+3. In the list of linked nodes, identify the node you want to unlink and click the **three-dot button** next to the IP
+   address of the node.
+
+4. Click **Unlink**.
+
+5. Click **Confirm**
+
+</TabItem>
+
+<TabItem value="On Follower Node">
+
+1. Log in to Local UI on the node you want to unlink.
+
+2. From the left **Main Menu**, click **Linked Edge Hosts**.
+
+3. Click **Unlink** in the upper-right corner of the **Linked Edge Hosts** page.
+
+4. Click **Confirm**
+
+</TabItem>
+
+</Tabs>
+
+### Validate
+
+1. Log in to Local UI on the node you unlinked.
+
+2. From the left **Main Menu**, click **Linked Edge Hosts**.
+
+3. Confirm that the node has been unlinked from the leader and is ready to be paired with another node.
+
+## Remove Leader Node Status
+
+If you have made a host the leader of a group of linked hosts, but later want to link it to another leader node, you
+must first remove its leader node status.
```

Comment:
This is kinda tripping me up because I thought all control-plane nodes were considered leaders?  🙈